pyecm.py

```diff
@@ -1,1483 +1,1569 @@
-#!/usr/bin/python3
-
-'''
-You should install psyco and gmpy if you want maximal speed.
+#!/usr/bin/python
```

Comment:
This should stay as `python3`, as many Linux distros still default to `python` --> `python2`.